From 2721fec289efe1f9faf66582b7a809eb70c3711b Mon Sep 17 00:00:00 2001
From: Daniel Leung <danielcp@gmail.com>
Date: Sat, 6 Jul 2019 14:24:58 -0700
Subject: [PATCH 1/1] xtensa.h needs stdint.h

Or else it complains about "uint32_t does not name a type".

Signed-off-by: Daniel Leung <danielcp@gmail.com>
---
 gdb/arch/xtensa.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gdb/arch/xtensa.h b/gdb/arch/xtensa.h
index 46572c9071..e0abdc0223 100644
--- a/gdb/arch/xtensa.h
+++ b/gdb/arch/xtensa.h
@@ -20,6 +20,8 @@
 #ifndef XTENSA_H
 #define XTENSA_H
 
+#include <stdint.h>
+
 /* Xtensa ELF core file register set representation ('.reg' section).
    Copied from target-side ELF header <xtensa/elf.h>.  */
 
-- 
2.21.0

